tests/amqpprox_session.t.cpp

```diff
@@ -365,6 +366,26 @@ void SessionTest::testSetupProxyOutOfOrderOpen(int idx)
     });
 }
 
+void SessionTest::testSetupProxyReceiveClose(int idx)
+{
+    // Client  <--------Close-----  Proxy  <--------Close------  Broker
+    methods::Close receivedClose;
+    receivedClose.setReply(123, "Broker is closing the connection");
+    d_clientState.pushItem(idx, Data(encode(receivedClose)));
```

Comment:
shouldn't this be server state?

Reply:
Basically, here amqpprox works as a client, when it receives Close method from server and later on when it forwards the method to client, it works as a server. So we are using the convention `d_clientState` to interact with server and `d_serverState` to interact with client. It is bit confusing. But that is the way all the tests are written. And I think the core code also has the same convention.